### PR TITLE
Bugfix/dot is missed and search failure/#2734

### PR DESCRIPTION
**Pull request by NewArtMask**
- Fixed dot missing at the end of the sentence after searching text which couldn't be found.

Before:
image

After:
image

- Fixed problem with search reauest. Changed url in getAllResults in search.service.ts to backEndLink.

### CODE REVIEW

**Comment by NewArtMask** [comment]
Fixed bugs:
[[Eco news] Not translated to Ukrainian/Russian with UA/RU localization #3164](https://github.com/ita-social-projects/GreenCity/issues/3164)
[[Eco news] Ukrainian and Russian localization, all words on the page are not translated #2696](https://github.com/ita-social-projects/GreenCity/issues/2696)

**Comment by NewArtMask** [comment]
Fixed bugs:
[[Search] The dot is missed at the end of the sentence after searching text which couldn't be found #2734](https://github.com/ita-social-projects/GreenCity/issues/2734)

File: src/app/main/component/layout/components/search-all-results/search-all-results.component.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="search-bar-wrapper">
     <div class="container search_bar-container">
       <img class="search_search-icon" [src]="searchIcons.search" alt="search icon" aria-hidden="true" />
+      <!-- input search value -->
```
Comment (rostikdoroshenko) [inline src/app/main/component/layout/components/search-all-results/search-all-results.component.html:5]: Comment?

Comment (NewArtMask) [reply src/app/main/component/layout/components/search-all-results/search-all-results.component.html:5]: It wasn't necessary.

Comment (NewArtMask) [reply src/app/main/component/layout/components/search-all-results/search-all-results.component.html]: Fixed! ;-)

**Review by nazarf1 — APPROVED** [review]

**Pull request closed without merging**